src/Infrastructure/Dto/Order/OrderResponse.php

```diff
@@ -70,7 +73,66 @@ public static function createFromOrder(Order $order): self
         foreach ($commitments as $commitment) {
             $self->printed += $commitment->getQuantity();
             $self->remaining -= $commitment->getQuantity();
-            $self->makers[] = $commitment->getMaker()->getId();
+        }
+
+        return $self;
+    }
+
+    public static function createFromOrderAndMaker(Order $order, Maker $maker): self
```

Comment:
bis auf wenige Zeilen ist das das gleiche wie createFromOrderAndRequester. das was hier redundant ist würde ich eine private function innerhalb der gleichen klasse auslagern und diese in beiden functionen aufrufen.